tutorials/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.conf.urls import include, url
+
+from tutorials import views
+
+urlpatterns = [
+    url(r'^$', views.ListTutorials.as_view(), name='list_tutorials'),
```

Comment:
The urls are there but they are not linked on the front page.

Reply:
That true since there is this new button on the home-page I will include the link here:
![image](https://user-images.githubusercontent.com/38939526/73740997-9c60dd80-4749-11ea-9d42-a52df4d7b17d.png)